rundeckapp/grails-app/services/rundeck/services/NotificationService.groovy

```diff
@@ -544,7 +545,7 @@ public class NotificationService implements ApplicationContextAware{
             userData['user.lastName'] = user.lastName
         }
         //pass data context
-        def dcontext = content['context']?.dataContext ?: [:]
+        def dcontext = content.context?.getSharedDataContext()?.getData(ContextView.global())?.getData() ?: [:] //usage of modified global context
```

Comment:
instead of adding to the shared data inside the plugin, can it just be consolidated here before getting the data?

Reply:
moved to the workflow operator since it was the original scope of the upvar plugin: https://github.com/rundeck/rundeck/pull/2952/files#diff-a3b56ffe2c1254af9525ee9c1f8ea5efR155